src/System.Text.Json/src/System/Text/Json/Linq/JObject.cs

```diff
@@ -6,42 +6,62 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace System.Text.Json
+namespace System.Text.Json.Linq
 {
     /// <summary>
     ///  Represents a mutable JSON object.
     /// </summary>
-    public sealed class JsonObject : JsonNode, IEnumerable<KeyValuePair<string, JsonNode>>
+    public sealed partial class JObject : JNode, IEnumerable<KeyValuePair<string, JNode>>
```

Comment:
I totally support a rename and a new namespace, but I don't think that in general clashing with Json.NET is a good idea.

Reply:
The renaming uniforms `System.Text.Json.Linq` with `System.Xml.Linq` which uses `X` prefixes instead of `Xml`  (: